test/integration/cli-test.coffee

```diff
@@ -4,6 +4,9 @@ express = require 'express'
 clone = require 'clone'
 bodyParser = require 'body-parser'
 fs = require 'fs'
+syncExec = require 'sync-exec'
```

Comment:
Also `sync-exec` says: "Upgrading to 0.12.x is usually safe. At that point it will use child_process.execSync." So ignore my comments, they're pointless!